### ISSUE TITLE
Square Instead of Circle (not an issue)

### ISSUE THREAD

**Issue by Fooot-Code** [Ryrden/perfect-circle#1]
Yo this is just a variation of the code, but if anyone wanted to, they could change the step increment to 90. This would draw a square instead of a circle, but it works just the same. I learned from [this Code Bullet video](https://www.youtube.com/watch?v=8jv8nOKR_Ck&ab_channel=CodeBulletsDayOff) that the way the Neal.Fun code works is by checking if points are equidistant from the center, and because pyautogui can draw the square so fast, Neal.fun thinks that the square is a circle, passing 99.99% almost all the time.

**Comment by Ryrden** [comment]
Hi! Thanks for the issue.

I checked the video and that's impressive how Neal.fun works, I didn't know about that. 

I tested here and is true but depends on how centered my mouse can be, using the already implemented method can reach 99,9% too.

![image](https://github.com/Ryrden/perfect-circle/assets/76923948/3f05cd5e-e845-44df-a600-378d29867d8f)

Well, I think wouldn't be nice to set the `step-increment` to 90 because technically wouldn't be more a circle but a note could be added on README referring to this mentioned video or this PR explaining that if the step is 90 a square will be drawn and scoring the max possible.

Feel free to open a pull request.
